hoover/search/es.py

```diff
@@ -1,17 +1,34 @@
+import json
+from contextlib import contextmanager
 from django.conf import settings
 from elasticsearch import Elasticsearch
 from elasticsearch.client.utils import _make_path
 from elasticsearch.exceptions import NotFoundError, RequestError, ConnectionError
 
-es = Elasticsearch(settings.HOOVER_ELASTICSEARCH_URL)
 DOCTYPE = 'doc'
 
 class SearchError(Exception):
     def __init__(self, reason):
         self.reason = reason
 
+@contextmanager
+def elasticsearch():
```

Comment:
I'd squash everything locally and create new commits. But if that's too much bother, we can just squash everything into one commit, feel free to do that.